packages/config/config/devices/0x015f/mh-s314.json

```diff
@@ -103,12 +134,13 @@
 			"allowManualEntry": true
 		},
 		"255": {
-			"label": "Factory setting",
+			"label": "Factory Setting",
 			"description": "85 restore factory setting",
```

Comment:
This doesn't make sense. 85 isn't allowed within the range. Could you verify? Usually we add an option that says Normal Operation and value 0 (that does nothing), then add an option with the reset value.